src/main/java/uk/gov/hmcts/reform/sscs/util/SscsUtil.java

```diff
@@ -0,0 +1,42 @@
+package uk.gov.hmcts.reform.sscs.util;
+
+import static org.apache.commons.lang3.StringUtils.equalsIgnoreCase;
+import static uk.gov.hmcts.reform.sscs.model.PartyItemList.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.hmcts.reform.sscs.ccd.domain.DynamicListItem;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+@Slf4j
+public class SscsUtil {
+
+    private SscsUtil() {
+        //
+    }
+
+    public static List<DynamicListItem> getPartiesOnCase(SscsCaseData sscsCaseData, boolean includeDwp, boolean includeHmcts) {
```

Comment:
Nice to have: This is quite a confusing api. Maybe have multiple different methods so we can remove the boolean parameters.

Reply:
Split this into 2 methods now